nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -243,15 +244,24 @@ def _dequantize(self, op, quantized_val):
     def quantize_weight(self, wrapper, **kwargs):
         config = wrapper.config
         module = wrapper.module
+        input = kwargs['input_tensor']
         weight = copy.deepcopy(wrapper.module.old_weight.data)
         weight_bits = get_bits_length(config, 'weight')
         quant_start_step = config.get('quant_start_step', 0)
         assert weight_bits >= 1, "quant bits length should be at least 1"
 
         # we dont update weight in evaluation stage
         if quant_start_step > self.bound_model.steps or not wrapper.training:
+            module.tracked_min_input, module.tracked_max_input = torch.min(input), torch.max(input)
```

Comment:
let's discuss this line. this line is only for initializing `module.tracked_min_input, module.tracked_max_input`? then we should think about how to initialize them.

Reply:
In current implementation, we initialize it during the process before `quant_start_step > self.bound_model.steps`. After this process, it will be updated by ema algorithm.